### PR TITLE
Update add-guests.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2312
Added NOTE

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by LolaJacobsen** [comment]
@somakbhattacharyya - please review.

File: Teams/add-guests.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ Anyone with a business or consumer email account, such as Outlook, Gmail, or oth
 
 As an admin, you can add a new guest user to the organization in a couple ways: 
 - Global admins who are owners of a team and owners of a team can add a guest to a team through either the Microsoft Teams desktop or the web clients. For more details, check out [Add guests to a team](https://support.office.com/article/add-guests-to-a-team-fccb4fa6-f864-4508-bdde-256e7384a14f)
+> [!NOTE] 
+> This does not apply when AAD admins and users in the guest inviter role can invite is enabled.
```
Comment (JohanFreelancer9) [inline Teams/add-guests.md]:
```suggestion
> This does not apply when **AAD admins and users in the guest inviter role can invite** is enabled.
```

File: Teams/add-guests.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ Anyone with a business or consumer email account, such as Outlook, Gmail, or oth
 
 As an admin, you can add a new guest user to the organization in a couple ways: 
 - Global admins who are owners of a team and owners of a team can add a guest to a team through either the Microsoft Teams desktop or the web clients. For more details, check out [Add guests to a team](https://support.office.com/article/add-guests-to-a-team-fccb4fa6-f864-4508-bdde-256e7384a14f)
+> [!NOTE] 
+> This does not apply when **AAD admins and users in the guest inviter role can invite** is enabled.
```
Comment (nenonix) [inline Teams/add-guests.md]:
```suggestion
> This does not apply when **Admins and users in the guest inviter role can invite** is enabled.
```

**Comment by scanum** [comment]
@somakbhattacharyya Changes look correct. Could you please review and merge? Thanks!

**Comment by LolaJacobsen** [comment]
I'm still working with @somakbhattacharyya on this. The note is out of context and doesn't list the workaround. We won't merge this PR until the PM Somak has agreed to the wording.

**Pull request merged**